tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/AlignedChunkMetadata.java

```diff
@@ -54,6 +55,14 @@ public Statistics getStatistics(int index) {
     return v == null ? null : v.getStatistics();
   }
 
+  public List<Statistics> getValueStatisticsList() {
+    List<Statistics> valueStatisticsList = new ArrayList<>();
+    for (IChunkMetadata v : valueChunkMetadataList) {
+      valueStatisticsList.add(v == null ? null : v.getStatistics());
+    }
+    return valueStatisticsList;
+  }
```

Comment:
I think it's better to add a method like `hasNullValue` in this class instead of doing the judgement outside.

Reply:
done